api/spec/smithy/model/types/Cluster.smithy

```diff
@@ -90,3 +90,13 @@ structure Metadata {
     version: String,
 }
 
+list Failures {
+    member: Failure
+}
+
+structure Failure {
+    @documentation("Failure code")
```

Comment:
nitpick, can we add some more details to this documentation annotations?

Reply:
Yes, expanded the documentation annotations to make it more clear